content/_partials/toptoolbar.html.haml

```diff
@@ -13,71 +13,67 @@
 
         %li.nav-item.dropdown
           .nav-link.dropdown-toggle{"aria-expanded" => "false", "aria-haspopup" => "true", "data-toggle" => "dropdown", :role => "button"}
-            Documentation
+            Docs
 
           .dropdown-menu
             %a.dropdown-item{:href => expand_link('doc')}
               Use Jenkins
             %a.dropdown-item{:href => expand_link('doc/developer')}
               Extend Jenkins
+            %span.dropdown-item
+              %strong
```

Comment:
Would it make sense to make 'top level' menu items always bold? Or apply a different formatting to the second-level items (which leading dash)?

Reply:
Second level items already have dashes added as you suggested. 😄

Reply:
To clarify, some top level items now are bold while others are not.

Reply:
Ah.  I only made items with children bold, but I see what you're saying.